pkger/service_test.go

```diff
@@ -821,24 +874,25 @@ func TestService(t *testing.T) {
 						},
 					},
 					{
-						name:    "guage",
+						name:    "with new name single stat",
```

Comment:
this one should be where it was previously, its a different test case than the ones that follow. It is testing the name change and the single stat, but the single stat is kind of a freebie here

Reply:
I did some more re-ordering/alphabetizing here. the tests for the gauge chart were moved up [here](https://github.com/influxdata/influxdb/blob/cc315a3dc6fceb3d0f7342efffc5c152d641c221/pkger/service_test.go#L787). lmk if anything seems off w/ the tests after I moved them